src/compiler/scala/tools/nsc/settings/Warnings.scala

```diff
@@ -29,9 +29,10 @@ trait Warnings {
       "Inspect both user-written code and expanded trees when generating unused symbol warnings."
     )
   )
-  val warnDeadCode         = BooleanSetting("-Ywarn-dead-code", "Warn when dead code is identified.")
-  val warnValueDiscard     = BooleanSetting("-Ywarn-value-discard", "Warn when non-Unit expression results are unused.")
-  val warnNumericWiden     = BooleanSetting("-Ywarn-numeric-widen", "Warn when numerics are widened.")
+  val warnDeadCode               = BooleanSetting("-Ywarn-dead-code", "Warn when dead code is identified.")
+  val warnValueDiscard           = BooleanSetting("-Ywarn-value-discard", "Warn when non-Unit expression results are unused.")
+  val warnNumericWiden           = BooleanSetting("-Ywarn-numeric-widen", "Warn when numerics are widened.")
```

Comment:
Please treat all boolean settings with respect. There's no need to shove them to the right. 😉

Reply:
The whole world is moving to the right.